report.ipynb

```diff
@@ -545,7 +546,7 @@
     "text.append(f\"\"\"\n",
     "<p>\n",
     "This report summarizes the detection of SARS-CoV-2 {variant_description} in sample \n",
-    "<strong>{os.path.basename(SAMPLE_PATH)}</strong> generated on the <strong>{INSTRUMENT_VENDOR}</strong> sequencing platform.\n",
+    "<strong>{os.path.basename(SAMPLE_PATH)}</strong>, generated on the <strong>{INSTRUMENT_VENDOR}</strong> sequencing platform with ARTIC V{ARTIC_PRIMER_VERSION} primers.\n",
```

Comment:
The value for `ARTIC_PRIMER_VERSION` comes from the software user, or from a default. As it reads now, there is an implicit assumption that it indicates what was actually used in the lab, but we don't know that since we're not (yet) inferring it from the data. 
"data analyzed for ARTIC primer set version" or even "data analyzed using <bedfile name / link> designed for `ARTIC_PRIMER_VERSION`" might be more agnostic. Thoughts?